notes/how to contribute to bqt documentation.md

```diff
@@ -1,22 +1,53 @@
-First, read the [[README]]
-
-## First time contributor, help?
-This will sound complex, and like a lot, but if you try step by step and google when you get stuck, or ask questions [here](https://github.com/techartorg/bqt/issues/92), you'll get there. 
-(Don't be afraid of making a mistake, you can't break the project if you're not admin.)
-Here we go 🚶:  
-- you will need to fork the repo with all branches
-- then clone the forked repo (big green button at the top on the front page)
-- swap branch to the wiki branch
-- open the folder in obsidian, and edit the text files (make a test change to anything)
-- make a single edit, push it to your fork
-- create a PR, and an admin should be able to review it. Once you successfully created a PR, feel free to close it. You are now ready to make real PRs.
-
-For your next steps, make actual changes to the docs.
-Then repeat the above steps to push the changes, and create a PR.
-Review it with a maintainer, maybe implement their feedback, and iterate on it until it get's merged in.
-
-## What should I work on
+Thank you for your interest in contributing to bqt's documentation! To get started, read the [[README]]. The following process may look complex, but we've broken each step down into bite-sized pieces. Don't be afraid to make mistakes - you can't break the project if you're not admin.
+
+### Prerequisites
+Before you start your first contribution, install the following tools:
+ - Git
+ - A Git client, like [Fork](https://git-fork.com/) or [GitHub Desktop](https://desktop.github.com/)
+ - [Obsidian](https://obsidian.md/)
+If you are new to using Git and GitHub, please read [about Git](https://docs.github.com/en/get-started/using-git/about-git).
+
+## Your first contribution
+The following steps are geared toward complete beginners. If you get lost or stuck, please feel free to ask questions [in this thread](https://github.com/techartorg/bqt/issues/92) or troubleshoot with Google search. Let's go! 🚶:  
+#### Fork the repository
+1. Navigate to [bqt's repository].(https://github.com/techartorg/bqt)
```

Comment:
small typo. the dot in between breaks link

Reply:
`[bqt's repository].(https://github.com/techartorg/bqt)`  
should be   
`[bqt's repository](https://github.com/techartorg/bqt)`